main.coffee: Updates coala's min version to 0.10.0

EverWinter23 opened the pull request:
update COALA_MIN_VERSION from '0.9.1' to '0.10.0'

alisianoi commented:
Hey, @EverWinter23 , you are not following [commit message guidelines][1]

In particular, please use imperative: `Updated` -> `Update`

Also, circleci appears to be failing because of some unrelated issue.

[1]: http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

EverWinter23 commented:
Should i keep working on it?

RohanVB commented:
Try to figure out what's causing circleci to fail on this build. 
When I pushed the same PR, ci passed. 
(https://github.com/coala/coala-atom/pull/44)

Also, fix the commit message. See: http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

RohanVB commented:
@EverWinter23 
Good job, it seemed to be a dependency issue. Can you please squash the extra commits? 
Also, `Updates` -> `Update` in the shortlog. 

You can also remove the indent tab in the commit body and have a new line between the issue reference and commit body so it complies to: http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

Other than that, lgtm :)

EverWinter23 commented:
Finally Done!!!

RohanVB commented on circle.yml:
```diff
@@ -3,17 +3,20 @@ dependencies:
 
     # Install coala
+    - pip install --upgrade pip
+    - pip install six
```
Is this required? Didn't your last build pass without `six`?
Also, the current passing build on master does work without upgrading pip, with v 6.x or something. Could you check if these 2 lines are required again please?

EverWinter23 replied:
i got six module not defined

EverWinter23 replied:
i'll try without that again

EverWinter23 replied:
cuz last time i hand't upgraded pip

RohanVB replied:
Sounds good. Better to spend an extra few minutes to verify :)

EverWinter23 replied:
Made changes.

EverWinter23 committed:
main.coffee: Update min coala version to '0.10.0'

Also resolves following atom dependecy issue:
Package libnotify64 not installed

Closes https://github.com/coala/coala-atom/issues/52

EverWinter23 commented:
Just upgrading pip works.

RohanVB commented:
ack 617d114141e1640e99744df8949ce69469226aac

meetmangukiya commented:
@RohanVB any reason this is not merged ?

RohanVB commented:
@meetmangukiya  [March 20, 2017 4:23 PM](https://gitter.im/coala/coala?at=58cfb4aff7f7d4810436f417)
(tldr; only members of the coala maintainers' group are allowed to merge)
